cfgov/jinja2/v1/_includes/organisms/mega-menu-var-1.html

```diff
@@ -136,6 +136,29 @@
             <div class="{{ _classes( nav_depth, '-lists' ) }}">
                 {{ _nav_list( nav_depth, nav_item.nav_groups ) }}
 
+                {% if flag_enabled( 'MEGA_MENU_BACKEND_V2' ) and nav_item.featured_links %}
```

Comment:
It looks like `other_links` will only be displayed if there are also `featured_links` -- I'm not sure if that's an intentional part of the design?

Reply:
Thanks, nice catch. Fixed in 634a09feb.